src/main/generic/consensus/base/account/tree/AccountsTreeNode.js

```diff
@@ -273,6 +273,26 @@ class AccountsTreeNode {
         return this._hash;
     }
 
+    /**
+     * Returns the number of children this node has
+     * @param {boolean} includeBranchNodes
```

Comment:
Use `[...]` to indicate optional parameters: `@param {boolean} [includeBranchNodes]`.